src/server/routes/api-routes.js

```diff
@@ -1,9 +1,10 @@
 import express from 'express';
 import items from '../items/routes';
+import validator from '../items/validator';
 
 const router = express.Router();
 
-router.use('/items', items);
+router.use('/items', validator, items);
```

Comment:
just for the record, 
we might be able to drag this validator out to  outer router.